### PR TITLE
Issue 10

### PR DESCRIPTION
**Pull request by Nerdenator**
### Summary
On lines 59-62 in src/terra-button.scss, added:

`&:-moz-focusring {
    outline: 2px solid -moz-mac-focusring;
    -moz-outline-radius: 0.45em;
}`

As per Brett Jankord's recommendation. Initial results were positive and the issue appears to be fixed on Mozilla Firefox 50/ OS X El Capitan. 

### Additional Details
Note that this was not tested on Ubuntu or Windows. Given the attribute name, it would be reasonable to think that any similar problems on those platforms in Firefox may not be addressed.

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -47,3 +47,9 @@ logs
 node_modules
 npm-debug.log*
 pids
+
+# for VS Code
```
Comment (gneatgeek) [inline .gitignore]: Please add these to your global gitignore: https://help.github.com/articles/ignoring-files/

File: src/terra-button.scss [hunk 2]
```diff
@@ -56,6 +56,11 @@ input[type='submit'].terra-Button {
     padding: 0;
   }
 
+  &:-moz-focusring {
```
Comment (gneatgeek) [inline src/terra-button.scss]: Typo: focusing

File: src/terra-button.scss [hunk 3]
```diff
@@ -56,6 +56,11 @@ input[type='submit'].terra-Button {
...
+    outline: 2px solid -moz-mac-focusring;
```
Comment (gneatgeek) [inline src/terra-button.scss]: Typo: focusing

File: dist/index.html [hunk 4]
```diff
@@ -0,0 +1,161 @@
+<!doctype html>
```
Comment (gneatgeek) [inline dist/index.html]: Please don't commit anything in the `dist` folder.

File: src/terra-button.scss (hunk 3 above)
Comment (bjankord) [inline src/terra-button.scss]: focusring is the correct spelling for this specific mozilla pseudo-class: https://developer.mozilla.org/en-US/docs/Web/CSS/:-moz-focusring

Comment (gneatgeek) [inline src/terra-button.scss]: Thanks for the clarification @bjankord!

**Comment by Nerdenator** [comment]
I have removed local gitignore modifications and removed dist folder output, the changes are reflected in commit [863125f](https://github.com/Nerdenator/terra-button/commit/863125f9378398be3f9fe7aced1c57607ed738c2) on my personal fork. Admittedly this is my first time implementing changes in a pull request; what is the next step on my part? Should I close out this request and create a new one on that commit?

**Comment by bjankord** [comment]
@Nerdenator You can keep this pull request open, it looks like it has all the changes needed fix the focus issue in Firefox. We'd like to get one more reviewer to review this before we merge the PR down to master. Thanks for the help with this @Nerdenator!

**Comment by Matt-Butler** [comment]
+1

**Pull request merged**